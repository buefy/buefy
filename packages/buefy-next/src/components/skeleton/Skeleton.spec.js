import { mount } from '@vue/test-utils'
import BSkeleton from '@components/skeleton/Skeleton'

let wrapper

describe('BSkeleton', () => {
    beforeEach(() => {
        wrapper = mount(BSkeleton)
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
