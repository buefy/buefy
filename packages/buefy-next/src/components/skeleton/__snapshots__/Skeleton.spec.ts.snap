// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BSkeleton > render correctly 1`] = `
"<div class=\\"b-skeleton is-animated\\">
  <div class=\\"b-skeleton-item is-rounded\\"></div>
</div>"
`;
